### ISSUE TITLE
Premium Content Block - Allow subscribers' access to be tracked

### ISSUE THREAD

**Issue by thabotswana** [Automattic/wp-calypso#50439]
### Problem
When a subscriber views or accesses Premium Content, there should be a record of that similar to how we collect Stats. If the subscriber disputes payment for the subscription, there is no way for the site owner to prove whether the user actually did access their content.

### Solution
Give site owners the ability to track their subscribers' views or when they access content like downloads. This would avoid subscribers being able to dispute charges claiming they never accessed the content.

An alternative would be to make this data available to HEs in the Blog RC or any other applicable logs.

**Additional context**
Slack discussion: p1614166537185400-slack-wpcom-support

**Comment by cathymcbride** [comment]
@apeatling would this be on the content side or the stats side?

**Comment by apeatling** [comment]
I think this would need both feature work and stat recording work. Is that what you're asking?

**Comment by cathymcbride** [comment]
Yes. I was thinking it would be one or the other, but it seems to be both. :) Would this be appropriate for the view backlog?
